docs/resources/cce_node.md

```diff
@@ -106,6 +106,74 @@ resource "huaweicloud_cce_node" "mynode" {
 }
 ```
 
+## Node with storage configuration
+
+```hcl
+resource "huaweicloud_cce_node" "mynode" {
+  cluster_id        = huaweicloud_cce_cluster.mycluster.id
+  name              = "mynode"
+  flavor_id         = "s3.large.2"
+  availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
+  key_pair          = huaweicloud_compute_keypair.mykp.name
+
+  root_volume {
+    size       = 40
+    volumetype = "SATA"
+  }
+  data_volumes {
+    size       = 100
+    volumetype = "SATA"
+  }
+
+  // Storage configuration
+  storage {
+    selectors {
+      name              = "cceUse"
+      type              = "evs"
+      match_label_size  = "100"
+      match_label_count = 1
+    }
+
+    selectors {
```

Comment:
can we provide one selector with `match_label_metadata_encrypted` for disk encrytion?

Reply:
sure, added